compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -33,10 +33,12 @@ class CompilationTests extends ParallelTesting {
 
   // @Test  // enable to test compileStdLib separately with detailed stats
   def compileStdLib: Unit = {
+    implicit val testGroup: TestGroup = TestGroup("compileStdLib")
```

Comment:
You shouldn't need to give an explicit type signature here because the implicit val is in a def.

Reply:
Do you want me to remove them? I added them to avoid warnings from the IDE.

Reply:
Your call.